src/Propel/Generator/Manager/ModelManager.php

```diff
@@ -194,4 +196,33 @@ protected function doBuild(AbstractOMBuilder $builder, $overwrite = true)
 
         return 1;
     }
+
+    /**
+     * Create script to import all table map files into database map.
+     *
+     * @return int
+     */
+    protected function createTableMapLoaderScript(): int
+    {
+        $schemas = $this->getDataModels();
+        $builder = new TableMapLoaderScriptBuilder($this->getGeneratorConfig());
+        $fileContent = $builder->build($schemas);
+
+        $file = $builder->getFile();
+        $filePath = $file->getPathname();
+
+        $action = $file->isFile() ? 'Updating' : 'Creating';
+        $this->log("\t-> $action $filePath");
+
+        if ($file->isFile() && file_get_contents($filePath) === $fileContent) {
+            $this->log("\t\t(no change)");
+
+            return 0;
```

Comment:
What does 0 mean - is that a console return code? maybe we can (re)use the class const CODE_SUCCESS / CODE_ERROR for those as more speaking ones instead of magic numbers?

Reply:
That's the number of changed files. It is collected by the calling method for user output. I'll add a description to the return parameter.